Makefile

```diff
@@ -166,8 +166,48 @@ oci-image-build: ## Build the aurae/auraed OCI images
 
 .PHONY: oci-run
 oci-run: ## Run the aurae/auraed OCI images
-	$(oci) run -v $(shell pwd):/app $(flags) $(tag) $(command)
+	$(oci) run -v $(shell pwd):/aurae $(flags) $(tag) $(command)
 
 .PHONY: oci-make
 oci-make: ## Run the makefile inside the aurae/auraed OCI images
-	$(oci) run -v $(shell pwd):/app --rm -it $(tag) $(command)
+	$(oci) run -v $(shell pwd):/aurae --rm -it $(tag) $(command)
+
+.PHONY: kernel
+kernel:
+	mkdir -p target/rootfs/boot
+	docker run -it --rm -u $${UID} -v "$(shell pwd):/aurae" aurae-kernel-builder bash -c "cd auraed/hack/kernel && ./mk-kernel"
+
+container-release:
```

Comment:
If you add a "description" to each of these Make targets it will be displayed when `make help` is ran. 

```make
target: other_target ## Description of target goes here after double pound sign
```